tests/test_sklearn_permutation_importance.py

```diff
@@ -9,11 +9,13 @@
 from sklearn.feature_selection import SelectFromModel
 from sklearn.linear_model import LogisticRegression
 
+from xgboost import XGBClassifier # type: ignore
+import pandas as pd # type: ignore
```

Comment:
xgboost and pandas are indeed available not in all test environments, to fix this it's possible to put the imports into the test function like this (see https://docs.pytest.org/en/latest/skipping.html#skipping-on-a-missing-import-dependency):

```
xgboost = pytest.importorskip('xgboost')
pd =  pytest.importorskip('pandas')
```

(also ``# type: ignore`` is not required here, as tests are not type-checked at the moment)

Reply:
fixed